### ISSUE TITLE
Fallback to uploaded color swatches when no simple product uploaded

### ISSUE THREAD

**Issue by alexdrupal** [tzyganu/Switcher#164]
Hello

Thank you for your amazing module.

I have the color attribute and I have uploaded custom images for color values. Worked well, but recently my customer asked if he can use images from associated products. I have changed config for this attribute and that worked as well.
But if there is no image uploaded for simple product it displays label.
Where I should modify your code so it fallback to display uploaded images. I mean those I've uploaded for my previous configuration for color attribute.

Long story short:
Simple product Image uploaded -> display it (works)
No simple image uploaded -> look for custom image uploaded for this attribute and if found, then display it. If not found -> fallback to label.

Thanks!

**Comment by tzyganu** [comment]
@alexdrupal This is not in the scope of the extension, but If you want to fork it at and take a swing at it, the images are retrieved from [this method](https://github.com/tzyganu/Switcher/blob/master/app/code/community/Easylife/Switcher/Block/Catalog/Product/View/Type/Configurable/Config.php#L384)

You can add an `else` statement to [this if statement](https://github.com/tzyganu/Switcher/blob/master/app/code/community/Easylife/Switcher/Block/Catalog/Product/View/Type/Configurable/Config.php#L391) where you can retrieve the images you manually uploaded.  
You will find the code that retrieves the uploaded images in [this `case`](https://github.com/tzyganu/Switcher/blob/master/app/code/community/Easylife/Switcher/Block/Catalog/Product/View/Type/Configurable/Config.php#L219)

**Comment by alexdrupal** [comment]
Thank you!
